trace_context/HTTP_HEADER_FORMAT.md

```diff
@@ -215,16 +220,26 @@ The `OWS` rule defines an optional whitespace. It is used where zero or more whi
 
 The `tracestate` field value is a `list` as defined below. The `list` is a series of `list-members` separated by commas `,`, and a `list-member` is a key/value pair separated by an equals sign `=`. Spaces and horizontal tabs surrounding `list-member`s are ignored. There can be a maximum of 32 `list-member`s in a `list`.
 
+Empty and whitespace-only list members are allowed. Libraries and
+platforms MUST accept empty `tracestate` headers, but SHOULD avoid
+sending them. The reason for allowing of empty list members in
+`tracestate` is a difficulty for implementor to recognize the empty
+value when multiple `tracestate` headers were sent. Whitespace
+characters are allowed for a similar reason as some frameworks will
+inject whitespace after `,` separator automatically even in case of an
+empty header.
```

Comment:
I understand the whitespace use case, but don't understand "empty list member" use case. Under what conditions would that occur?

Reply:
We tried python, C# and Go. And there are libraries which will merge headers for you instead of giving the list. Also some libraries will insert spaces with the comma, some wouldn't. So make spec loose to simplify implementors life

Reply:
if libraries join headers into a string, they may add whitespace, but they will not create empty list members unless the inbound headers have empty values, which I think is not allowed by HTTP spec.

Reply:
We discussed not defining trace-state as a list. Technically, this FORBIDS an implementation to split the header into multiple headers.